tests/subsys/bluetooth/fast_pair/crypto/src/main.c

```diff
@@ -169,14 +170,136 @@ static void test_bloom_filter(void)
 			  "Invalid resulting filter.");
 }
 
+static void test_aes_ctr(void)
```

Comment:
Please arrange the testing order according to the fmn_crypto.h

1. sha256
2. hmac_sha256
3. aes128
4. aes128_ctr
(...)